### PR TITLE
cmd/observe: Add --first to support querying for earlier flows and events

### PR DESCRIPTION
**Pull request by chancez**
Depends on https://github.com/cilium/cilium/pull/19819

### CODE REVIEW

**Comment by chancez** [comment]
@gandro I was waiting for a new release branch to be cut that has my changes so I can update the reference to vendor. Do we have a branch I can update the vendored cilium/cilium to yet?

**Comment by rolinh** [comment]
@chancez No need for a new release branch to be cut. Just pick latest master, we'll take care of updating the dependency to the next RC once it's out.

**Comment by chancez** [comment]
@gandro @rolinh ok I rebased & updated the vendored cilium dependency to use master. PTAL.

**Comment by chancez** [comment]
How I tested:

I ran the following two commands at "roughly the same time" and they typically contained the exact same 5 flows (if I ran them "at the same time").
```
./hubble observe --all | head -n 5
```

```
./hubble observe --first 5
```

**Review by gandro — APPROVED** [review]
Perfect, thanks!

File: cmd/observe/flows.go [hunk 1]
```diff
@@ -198,6 +198,7 @@ more.`,
 	selectorFlags := pflag.NewFlagSet("selectors", pflag.ContinueOnError)
 	selectorFlags.BoolVar(&selectorOpts.all, "all", false, "Get all flows stored in Hubble's buffer")
 	selectorFlags.Uint64Var(&selectorOpts.last, "last", 0, fmt.Sprintf("Get last N flows stored in Hubble's buffer (default %d)", defaults.FlowPrintCount))
+	selectorFlags.Uint64Var(&selectorOpts.first, "first", 0, fmt.Sprintf("Get last N flows stored in Hubble's buffer (default %d)", defaults.FlowPrintCount))
```
Comment (rolinh) [inline cmd/observe/flows.go:201]:
```suggestion
	selectorFlags.Uint64Var(&selectorOpts.first, "first", 0, fmt.Sprintf("Get first N flows stored in Hubble's buffer (default %d)", defaults.FlowPrintCount))
```

**Review by rolinh — CHANGES_REQUESTED** [review]
LGTM except that copy paste type in the flag help message.

File: cmd/observe/flows.go (hunk 1 above)
Comment (chancez) [reply cmd/observe/flows.go]: Fixed

**Review by rolinh — APPROVED** [review]

**Pull request merged**